### ISSUE TITLE
How to disable MathJax?

### ISSUE THREAD

**Issue by erosennin** [sphinx-doc/sphinx#5568]
### Problem
Recent versions of Sphinx unconditionally include `MathJax.js` in HTML files, even if `sphinx.ext.mathjax` is not enabled in `conf.py`:

```
<script async="async" type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
```

This can be an issue for projects that do not use maths and do not need a math renderer.

Even with `html_math_renderer = None` or `html_math_renderer = 'imgmath'` in `conf.py`, `MathJax.js` is still there.

Is there a way to disable loading `MathJax.js`?

Thanks. :)

#### Procedure to reproduce the problem
1. Create a new project with `sphinx-quickstart`.
2. Answer `no` when asked if you want to enable MathJax.
3. `make html`
4. `grep -i mathjax _build/html/index.html`

### Environment info
- OS: Linux
- Python version: 3.6.6
- Sphinx version: 1.8.1

**Comment by mitya57** [comment]
This is already fixed in #5536.

**Comment by erosennin** [comment]
Great, thanks!
